src/vm/amd64/profiler.cpp

```diff
@@ -41,6 +41,11 @@ typedef struct _PROFILE_PLATFORM_SPECIFIC_DATA
     UINT64      flt5;
     UINT64      flt6;
     UINT64      flt7;
+    UINT64      rdi;
```

Comment:
@janvorli - On Arm32, Jit32 prespills arguments on to stack in the prolog of the method. See `genProfilingEnterCallback()` for Arm32 and Legacy_backend case.  I am pasting the relevent comment block from that routine below:

 ```
   // On Arm arguments are prespilled on stack, which frees r0-r3.
    // For generating Enter callout we would need two registers and one of them has to be r0 to pass profiler handle.
    // The call target register could be any free register.
```